### ISSUE TITLE
Greetings from gitlab neoexpert JVM!

### ISSUE THREAD

**Issue by neo-expert** [guxingke/mini-jvm#16]
Hi, you wrote an issue on my project on gitlab (Metacircular JVM in pure Java https://gitlab.com/neoexpert/jvm ). I wisch you success in your project too :-)
maybe we can cooperate. Maybe your JVM could run on my JVM and vice versa. It woult be awesome.

**Comment by guxingke** [comment]
yeah , i can not agree more.  but my jvm like a baby compare for yours . could you some suggest for me? 
---
i will try run my jvm on yours , Wonderful journey absolutely.

**Comment by guxingke** [comment]
```java -jar jvm-core/target/mini-jvm.jar -jar jvm-core/target/mini-jvm.jar -jar test.jar ```

Implements the basic metacircular jvm already. 

thank you,  this benefit a lot from your project.
